packages/handlersjs-http/lib/models/http-handler-route.ts

```diff
@@ -45,6 +45,8 @@ export abstract class HttpHandlerOperation {
     public description?: string,
     public responses?: HttpHandlerOperationResponse[],
     public security?: HttpHandlerOperationSecurityType,
+    public addDateHeader?: boolean,
```

Comment:
```suggestion
    public addDate?: boolean,
```

Maybe just name this shorter, since we also not meantin 'header' in the other new attributes.